CSharpier 0.30.0 fails in Rider extension due to name change

<!--

BEFORE SUBMITTING AN ISSUE:

Option requests are out of scope for CSharpier, they will be closed without discussion.

See [Editor Troubleshooting](https://csharpier.com/docs/EditorsTroubleshooting) for how to view logs and troubleshoot some issues with csharpier not installing/running.

Note that the extension is versioned separately from CSharpier. It is designed to work with any version of CSharpier.

-->

**Environments**
- IDE Version:
- Extension Version: 
- CSharpier Version:  
*will NOT be the same as the extension version*
- Operating System:
- .csharpierrc Settings:
- .editorconfig Settings:

**Log Output**
```
2024-11-18 11:08:50,698 [  27867] SEVERE - #c.i.c.CSharpierLogger - JetBrains Rider 2024.3  Build #RD-243.21565.191
2024-11-18 11:08:50,698 [  27867] SEVERE - #c.i.c.CSharpierLogger - JDK: 21.0.5; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-11-18 11:08:50,698 [  27867] SEVERE - #c.i.c.CSharpierLogger - OS: Windows 11
2024-11-18 11:08:50,698 [  27867] SEVERE - #c.i.c.CSharpierLogger - Plugin to blame: CSharpier version: 1.8.2
2024-11-18 11:08:50,699 [  27868] SEVERE - #c.i.c.CSharpierLogger - Last Action: SaveAll
2024-11-18 11:08:50,699 [  27868] SEVERE - #c.i.c.CSharpierLogger - Current Command: Typing
2024-11-18 11:08:50,699 [  27868]   FINE - #c.i.c.CSharpierLogger - Removing directory at C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0 because it appears to be corrupted
2024-11-18 11:08:50,728 [  27897]   FINE - #c.i.c.CSharpierLogger - Running C:\Program Files\dotnet\dotnet.exe tool install csharpier --version 0.30.0 --tool-path C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0
2024-11-18 11:08:53,209 [  30378]   FINE - #c.i.c.CSharpierLogger - Running C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0\dotnet-csharpier --version in C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0
2024-11-18 11:08:53,209 [  30378] SEVERE - #c.i.c.CSharpierLogger - Cannot run program "C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0\dotnet-csharpier" (in directory "C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0"): CreateProcess error=2, The system cannot find the file specified
java.io.IOException: Cannot run program "C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0\dotnet-csharpier" (in directory "C:\Users\LukeParker\AppData\Local\CSharpier\0.30.0"): CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
	at com.intellij.csharpier.ProcessHelper.executeCommand(ProcessHelper.java:34)
	at com.intellij.csharpier.CustomPathInstaller.validateInstall(CustomPathInstaller.java:66)
	at com.intellij.csharpier.CustomPathInstaller.ensureVersionInstalled(CustomPathInstaller.java:58)
	at com.intellij.csharpier.CSharpierProcessProvider.setupCSharpierProcess(CSharpierProcessProvider.java:250)
	at com.intellij.csharpier.CSharpierProcessProvider.findAndWarmProcess(CSharpierProcessProvider.java:98)
	at com.intellij.csharpier.CSharpierProcessProvider.documentChanged(CSharpierProcessProvider.java:67)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at com.intellij.util.EventDispatcher.dispatchVoidMethod(EventDispatcher.java:120)
	at com.intellij.util.EventDispatcher.lambda$createMulticaster$1(EventDispatcher.java:85)
	at jdk.proxy2/jdk.proxy2.$Proxy114.documentChanged(Unknown Source)
	at com.intellij.openapi.editor.impl.DocumentImpl.lambda$changedUpdate$4(DocumentImpl.java:946)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeNonCancelableSection$3(CoreProgressManager.java:283)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:749)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$computeInNonCancelableSection$4(CoreProgressManager.java:291)
	at com.intellij.openapi.progress.Cancellation.computeInNonCancelableSection(Cancellation.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeInNonCancelableSection(CoreProgressManager.java:291)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeNonCancelableSection(CoreProgressManager.java:282)
	at com.intellij.openapi.editor.impl.DocumentImpl.changedUpdate(DocumentImpl.java:943)
	at com.intellij.openapi.editor.impl.DocumentImpl.updateText(DocumentImpl.java:847)
	at com.intellij.openapi.editor.impl.DocumentImpl.insertString(DocumentImpl.java:574)
	at com.intellij.openapi.editor.EditorModificationUtilEx.insertStringAtCaretNoScrolling(EditorModificationUtilEx.java:79)
	at com.intellij.openapi.editor.EditorModificationUtilEx.insertStringAtCaret(EditorModificationUtilEx.java:55)
	at com.intellij.openapi.editor.EditorModificationUtilEx.insertStringAtCaret(EditorModificationUtilEx.java:47)
	at com.jetbrains.rider.editorActions.AsynchronousFrontendTypedHandlerStrategy.K(AsynchronousFrontendTypedHandlerStrategy.kt:23)
	at com.jetbrains.rider.editorActions.AsynchronousFrontendTypedHandlerStrategy.access$optimisticTypeHandler(AsynchronousFrontendTypedHandlerStrategy.kt:11)
	at com.jetbrains.rider.editorActions.AsynchronousFrontendTypedHandlerStrategy$typeChar$1.invoke(AsynchronousFrontendTypedHandlerStrategy.kt:17)
	at com.jetbrains.rider.editorActions.AsynchronousFrontendTypedHandlerStrategy$typeChar$1.invoke(AsynchronousFrontendTypedHandlerStrategy.kt:17)
	at com.jetbrains.rider.editorActions.TypingSessions.K(TypingSessions.kt:263)
	at com.jetbrains.rider.editorActions.TypingSessions.type(TypingSessions.kt:221)
	at com.jetbrains.rider.editorActions.AsynchronousFrontendTypedHandlerStrategy.typeChar(AsynchronousFrontendTypedHandlerStrategy.kt:17)
	at com.jetbrains.rider.editorActions.FrontendTypedHandlerManager.typeCharOnBackend(FrontendTypedHandlerManager.kt:52)
	at com.jetbrains.rider.editorActions.FrontendTypedHandlerDelegate.beforeSelectionRemoved(FrontendTypedHandlerDelegate.kt:25)
	at com.intellij.codeInsight.editorActions.TypedHandler.callDelegates(TypedHandler.java:269)
	at com.intellij.codeInsight.editorActions.TypedHandler.lambda$doExecute$1(TypedHandler.java:187)
	at com.intellij.openapi.editor.impl.CaretModelImpl.lambda$runForEachCaret$3(CaretModelImpl.java:304)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:413)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:313)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:290)
	at com.intellij.codeInsight.editorActions.TypedHandler.doExecute(TypedHandler.java:158)
	at com.intellij.codeInsight.editorActions.TypedHandler.execute(TypedHandler.java:136)
	at com.intellij.codeInsight.lookup.impl.LookupTypedHandler.execute(LookupTypedHandler.java:75)
	at com.intellij.codeInsight.template.emmet.EmmetPreviewTypedHandler.execute(EmmetPreviewTypedHandler.java:43)
	at com.intellij.openapi.editor.impl.DefaultRawTypedHandler$1.run(DefaultRawTypedHandler.java:55)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction$lambda$5(AnyThreadWriteThreadingSupport.kt:379)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:389)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:379)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:896)
	at com.intellij.openapi.editor.impl.DefaultRawTypedHandler.execute(DefaultRawTypedHandler.java:49)
	at com.jetbrains.rdclient.editorActions.cwm.FrontendAsyncRawTypedHandler.execute(FrontendAsyncRawTypedHandler.kt:71)
	at com.intellij.openapi.editor.impl.MyRawTypedHandler.execute(EditorFactoryImpl.kt:290)
	at com.jetbrains.rider.editorActions.RiderRawTypedHandler.execute(RiderRawTypedHandler.kt:13)
	at com.intellij.openapi.editor.actionSystem.TypedAction.lambda$actionPerformed$2(TypedAction.java:201)
	at com.intellij.reporting.FreezeLoggerImpl.runUnderPerformanceMonitor(FreezeLoggerImpl.java:30)
	at com.intellij.openapi.editor.actionSystem.TypedAction.actionPerformed(TypedAction.java:200)
	at com.intellij.openapi.editor.impl.EditorImpl.processKeyTypedNormally(EditorImpl.java:1402)
	at com.intellij.openapi.editor.impl.EditorImpl.processKeyTyped(EditorImpl.java:1384)
	at com.intellij.openapi.editor.impl.EditorImpl.processKeyTyped(EditorImpl.java:3777)
	at com.intellij.openapi.editor.impl.EditorImpl$6.keyTyped(EditorImpl.java:1216)
	at java.desktop/java.awt.AWTEventMulticaster.keyTyped(AWTEventMulticaster.java:247)
	at java.desktop/java.awt.AWTEventMulticaster.keyTyped(AWTEventMulticaster.java:247)
	at java.desktop/java.awt.AWTEventMulticaster.keyTyped(AWTEventMulticaster.java:247)
	at java.desktop/java.awt.Component.processKeyEvent(Component.java:6617)
	at java.desktop/javax.swing.JComponent.processKeyEvent(JComponent.java:2901)
	at java.desktop/java.awt.Component.processEvent(Component.java:6439)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5032)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
	at java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1954)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:886)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1162)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1023)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:849)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent$lambda$1(IdeKeyboardFocusManager.kt:48)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent$lambda$2(IdeKeyboardFocusManager.kt:54)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:1054)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1054)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent(IdeKeyboardFocusManager.kt:54)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4909)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2810)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:675)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.kt:609)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$22(IdeEventQueue.kt:565)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:565)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16$lambda$15(IdeEventQueue.kt:355)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:857)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16(IdeEventQueue.kt:354)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2$lambda$1(IdeEventQueue.kt:1045)
	at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:916)
	at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
	at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2(IdeEventQueue.kt:1045)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:1054)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:117)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1054)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:395)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:500)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:159)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
	... 123 more
2024-11-18 11:08:53,210 [  30379] SEVERE - #c.i.c.CSharpierLogger - JetBrains Rider 2024.3  Build #RD-243.21565.191
2024-11-18 11:08:53,210 [  30379] SEVERE - #c.i.c.CSharpierLogger - JDK: 21.0.5; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-11-18 11:08:53,210 [  30379] SEVERE - #c.i.c.CSharpierLogger - OS: Windows 11
2024-11-18 11:08:53,210 [  30379] SEVERE - #c.i.c.CSharpierLogger - Plugin to blame: CSharpier version: 1.8.2
2024-11-18 11:08:53,210 [  30379] SEVERE - #c.i.c.CSharpierLogger - Last Action: SaveAll
2024-11-18 11:08:53,210 [  30379] SEVERE - #c.i.c.CSharpierLogger - Current Command: Typing
2024-11-18 11:08:53,297 [  30466]   FINE - #c.i.c.CSharpierLogger - beforeDocumentSaving for DocumentImpl[AdminNavigation.razor.cs]
```
**Steps to reproduce**
![image](https://github.com/user-attachments/assets/3613ed90-2abf-4800-882f-2b37b6c3ae85)
![image](https://github.com/user-attachments/assets/e795297e-6f45-4761-9c66-43056c0724a3)

**Expected behavior**

**Actual behavior**

--- a/Src/CSharpier.Cli/CSharpier.Cli.csproj
+++ b/Src/CSharpier.Cli/CSharpier.Cli.csproj
@@ -6,7 +6,6 @@
     <AssemblyName>dotnet-csharpier</AssemblyName>
     <TargetFrameworks>net8.0;net9.0</TargetFrameworks>
     <PackAsTool>true</PackAsTool>
-    <ToolCommandName>csharpier</ToolCommandName>
     <AssemblyOriginatorKeyFile>../../Nuget/csharpier.snk</AssemblyOriginatorKeyFile>
     <RollForward>Major</RollForward>
     <SignAssembly>True</SignAssembly>

